src/utils/safeTransactions.ts

```
@@ -18,11 +19,46 @@ import { web3 } from '@hoprnet/hopr-sdk';
 /** Human readable explanation of what the transaction is going to do */
 export const getRequestOfPendingTransaction = (transaction: SafeMultisigTransactionResponse) => {
   if (transaction.data) {
+    const safeAddress = transaction.safe;
+    const dataDecoded = transaction?.dataDecoded as any;
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)